diff -ur libwww-perl-5.53.orig/lib/LWP/Authen/Digest.pm libwww-perl-5.53/lib/LWP/Authen/Digest.pm
--- libwww-perl-5.53.orig/lib/LWP/Authen/Digest.pm	Tue Jan 23 03:41:01 2001
+++ libwww-perl-5.53/lib/LWP/Authen/Digest.pm	Tue May  1 00:36:59 2001
@@ -1,7 +1,7 @@
 package LWP::Authen::Digest;
 use strict;
 
-require MD5;
+require Digest::MD5;
 
 sub authenticate
 {
@@ -18,7 +18,7 @@
     my $uri = $request->url->path_query;
     $uri = "/" unless length $uri;
 
-    my $md5 = new MD5;
+    my $md5 = new Digest::MD5;
 
     my(@digest);
     $md5->add(join(":", $user, $auth_param->{realm}, $pass));
