diff -urN libwww-perl-5.50.orig/bin/lwp-request.PL libwww-perl-5.50/bin/lwp-request.PL
--- libwww-perl-5.50.orig/bin/lwp-request.PL	Sun Feb 11 18:30:04 2001
+++ libwww-perl-5.50/bin/lwp-request.PL	Sun Feb 11 18:39:11 2001
@@ -441,18 +441,15 @@
     if ($response->is_success) {
 	unless ($options{'d'}) {
 	    if ($options{'o'} &&
-		$response->content_type eq 'text/html') {
-		require HTML::Parse;
+		$response->content_type eq 'text/html') { require HTML::Parse;
 		my $html = HTML::Parse::parse_html($response->content);
 		{
-		    $options{'o'} eq 'ps' && do {
-			require HTML::FormatPS;
+		    $options{'o'} eq 'ps' && do { require HTML::FormatPS;
 			my $f = HTML::FormatPS->new;
 			print $f->format($html);
 			last;
 		    };
-		    $options{'o'} eq 'text' && do {
-			require HTML::FormatText;
+		    $options{'o'} eq 'text' && do { require HTML::FormatText;
 			my $f = HTML::FormatText->new;
 			print $f->format($html);
 			last;
diff -urN libwww-perl-5.50.orig/lib/LWP/Protocol/GHTTP.pm libwww-perl-5.50/lib/LWP/Protocol/GHTTP.pm
--- libwww-perl-5.50.orig/lib/LWP/Protocol/GHTTP.pm	Sun Feb 11 18:30:04 2001
+++ libwww-perl-5.50/lib/LWP/Protocol/GHTTP.pm	Sun Feb 11 18:39:44 2001
@@ -17,9 +17,7 @@
 @ISA=qw(LWP::Protocol);
 
 require HTTP::Response;
-require HTTP::Status;
-
-use HTTP::GHTTP qw(METHOD_GET METHOD_HEAD METHOD_POST);
+require HTTP::Status; use HTTP::GHTTP qw(METHOD_GET METHOD_HEAD METHOD_POST);
 
 my %METHOD =
 (
diff -urN libwww-perl-5.50.orig/lib/LWP/Protocol/https.pm libwww-perl-5.50/lib/LWP/Protocol/https.pm
--- libwww-perl-5.50.orig/lib/LWP/Protocol/https.pm	Sun Feb 11 18:30:04 2001
+++ libwww-perl-5.50/lib/LWP/Protocol/https.pm	Sun Feb 11 18:38:20 2001
@@ -11,8 +11,7 @@
     $SSL_CLASS = "IO::Socket::SSL"; # it was already loaded
 } else {
     eval { require Net::SSL; };     # from Crypt-SSLeay
-    if ($@) {
-	require IO::Socket::SSL;
+    if ($@) {require IO::Socket::SSL;
 	$SSL_CLASS = "IO::Socket::SSL";
     } else {
 	$SSL_CLASS = "Net::SSL";
diff -urN libwww-perl-5.50.orig/lib/LWP/Protocol/mailto.pm libwww-perl-5.50/lib/LWP/Protocol/mailto.pm
--- libwww-perl-5.50.orig/lib/LWP/Protocol/mailto.pm	Sun Feb 11 18:30:04 2001
+++ libwww-perl-5.50/lib/LWP/Protocol/mailto.pm	Sun Feb 11 18:40:18 2001
@@ -52,9 +52,7 @@
 				  "LWP::file::request called for '$scheme'";
     }
     if ($^O eq "MacOS") {
-	eval {
-	    require Mail::Internet;
-	};
+	eval { require Mail::Internet; };
 	if($@) {
 	    return new HTTP::Response &HTTP::Status::RC_INTERNAL_SERVER_ERROR,
 	               "You don't have MailTools installed";
